src/dbally/iql_generator/prompt.py

```diff
@@ -53,10 +54,12 @@ def __init__(
             question: Question to be asked.
             filters: List of filters exposed by the view.
             examples: List of examples to be injected into the conversation.
+            aggregations: List of examples to be computed for the view (Currently 1 aggregation supported).
```

Comment:
The same comment as the one for `IQLGenerator` + "(Currently 1 aggregation supported)" sounds like you can only pass a single aggregation here, which as far as I understand is not true.